extractors/python_functions/zipcode_extraction/code_snippet.md

```diff
@@ -1,19 +1,19 @@
 ```python
 import re
 
-YOUR_ATTRIBUTE = "text" # Choose any attribute here.
-COUNTRY_ID = "US" # Or CA, DE. Visit GitHub for extensive list of country zip-codes
+YOUR_ATTRIBUTE: str = "text" # Choose any attribute here.
+COUNTRY_ID: str = "US" # Or CA, DE. Visit GitHub for extensive list of country zip-codes
+YOUR_LABEL: str = f"{COUNTRY_ID} zip code"
```

Comment:
this wouldn't be parsed correctly -> i can only decide on one label atm

Reply:
Alright